test/vic-v2/00-all-fields.e2e.spec.js

```diff
@@ -45,21 +47,84 @@ const runTest = E2eHelpers.createE2eTest(
         .click('.form-panel .usa-button-primary');
       E2eHelpers.expectNavigateAwayFrom(client, '/contact-information');
 
+      // Photo page
+      client.assert.cssClassPresent('.progress-bar-segmented div.progress-segment:nth-child(3)', 'progress-segment-complete');
+      client.waitForElementVisible('label.usa-button.usa-button-secondary', Timeouts.normal);
+      client.axeCheck('.main');
+
+      // Disable upload button style to reveal input for test
+      /* HACK: style overridden so browser driver can find/interact with hidden inputs
+         (see https://github.com/nightwatchjs/nightwatch/issues/505) */
```

Comment:
Is this comment and/or the use of the HACK tag helpful?

Reply:
I think so- It's a good idea to document anything that changes the system under test.

Reply:
Thanks for the helpful guidance, that's a good way to determine whether to use these.